### PR TITLE
feat_: endpoint for testing panic reports

### PR DESCRIPTION
**Pull request by igor-sirotin**
Simple endpoint for testing Sentry integration.
Used a custom `intendedPanic` error type for simple distinguishing in Sentry:

image

https://sentry.infra.status.im/organizations/sentry/issues/135/?project=7&query=&statsPeriod=30d&stream_index=0

The `message` argument is there for simplicity of searching specific event just in case.

### CODE REVIEW

File: mobile/status.go [hunk 1]
```diff
@@ -2261,3 +2261,13 @@ func addCentralizedMetric(requestJSON string) string {
 
 	return metric.ID
 }
+
+func Panic(message string) string {
```
Comment (osmaczko) [inline mobile/status.go]:
```suggestion
func IntendedPanic(message string) string {
```

**Review by osmaczko — COMMENTED** [review]
Does it need to be merged? :thinking:

File: mobile/status.go (hunk 1 above)
Comment (igor-sirotin) [reply mobile/status.go]: Had the same in mind, but was unsure.
Let's make it `IntendedPanic` indeed then.

**Comment by igor-sirotin** [comment]
> Does it need to be merged? 🤔

@osmaczko This endpoint here doesn't hurt to exist, so I plan to merge it.
There will also be a desktop and mobile PRs providing a button which calls this endpoint. And that 2 PRs won't be merged.

The idea is to provide a way to check that Release CI builds properly work with Sentry.

**Comment by ilmotta** [comment]
> Does it need to be merged? 🤔

Today I actually had a need for such an endpoint because I needed to force panics in a reliable way during development, and while pointing to the cloud Sentry instance I'm using for testing purposes.

**Review by ilmotta — APPROVED** [review]

**Review by osmaczko — APPROVED** [review]
Okay then, let it land :earth_africa:

**Pull request merged**